hazelcast/test/src/HazelcastTests6.cpp

```diff
@@ -1115,6 +1115,26 @@ namespace hazelcast {
 
                 }
 
+                TEST_P(MixedMapAPITest, testTryLockTtl) {
```

Comment:
MixedMap API will be deleted in the next PR (since there will be no need for this, we will be supporting it in a different way) I will submit, hence you may not do this at all, since it will be deleted anyway.